docs-compose/overwatch_docs/content/DeployOverwatch/SystemTableIntegration/SystemTableConfiguration.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "System Table Configuration"
+date: 2024-01-23T16:30:21+05:30
+---
+
+## Configuration changes required for System Table Integration
+
+There is no unique configuration process for System Table ; however, the format for the **BELOW CONFIGURATIONS**
+are specific for System Table Integration. For all other configurations, please follow the
+[Configuration]({{%relref "DeployOverwatch/ConfigureOverwatch/Configuration.md"%}})
+
+* **auditlogprefix_source_path** - Instead of adding a fully qualified path (s3 or GC) for auditlog, 
+add keyword **system** in this column. This will enable the system table integration.
+
+**Same congifuration changes can be used for migrating from legacy to system table integration.**
+
+{{% notice note %}}
+For Azure deployments since Eventhub is not required for System Table Integration, we can leave these 
+fields blank - eh_name, Event hub name, eh_name, eh_conn_string, aad_tenant_id, aad_client_id, aad_client_secret_key, 
+aad_authority_endpoint
+{{% /notice %}}
+
+## Configuration changes required for Multi Account System Table Integration
+The process for achieving multi-account deployment is straightforward. In the workspace of each additional account, 
+a new sql_endpoint/warehouse needs to be created. The endpoint details are then passed to Overwatch through the configuration, 
```

Comment:
we don't actually need customers to create a new endpoint just for this. We can say we require a sql endpoint, and encourage them to use serverless since that would be the best approach

Reply:
done. change the statement to create new or use an existing warehouse